spec/data-model/README.md

```diff
@@ -78,13 +78,13 @@ type Message = PatternMessage | SelectMessage;
 
 interface PatternMessage {
   type: "message";
-  declarations: Declaration[];
+  declarations: (Declaration | UnsupportedStatement)[];
```

Comment:
Could the `declarations` field be renamed to something else? I'm not sure what, but it seems confusing to me that a list called "declarations" can include something that's not a declaration.